docs/business-webapp/broadcast/get_subscribers/telegram.mdx

```diff
@@ -0,0 +1,17 @@
+---
+title: Get subscribers on Telegram
+sidebar_position: 2
+sidebar_label: Telegram
+---
+
+To obtain Subscribers via Telegram, it's essential to have the Hermes Protocol bot added to your community. Start by opening a chat with our bot through [this link](https://t.me/HermesProtocolBot), and then open its Telegram Profile and choose "Add to Group".
+
+![create list](/img/broadcast/telegram-auth.png)
+
+:::info
+You can only invite the bot to a Group where you are an Admin.
+:::
+
+Once the bot is successfully added to your community, use the following command to generate the subscribe List embed:
+
+... (work in progress) ...
```

Comment:
Perhaps set this page as a draft to not be built.

```mdx
---
title: Get subscribers on Telegram
sidebar_position: 2
sidebar_label: Telegram
draft: true
---
```

Reply:
Left it without so it doesn't break links